charts/generic-service/values.yaml

```
@@ -216,8 +216,8 @@ postgresDatabaseRestore:
 
 scheduledDowntime:
   enabled: false
-  # Start at 6am UTC Monday-Friday
-  startup: '0 6 * * 1-5'
+  # Start at 6:45am UTC Monday-Friday
+  startup: '45 6 * * 1-5'
```

This would be 7.45am in the summer, which would mean that if anyone was working around that time then their environments wouldn't be up.  How about 6.30am and see how we get on?

```suggestion
  # Start at 6:30am UTC Monday-Friday
  startup: '30 6 * * 1-5'
```